osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs

```diff
@@ -588,6 +589,65 @@ public void TestAutoplayViaCtrlEnter()
             AddAssert("mod disabled", () => songSelect.Mods.Value.Count == 0);
         }
 
+        [Test]
+        public void TestCinemaViaCtrlShiftEnter()
+        {
+            addRulesetImportStep(0);
+
+            createSongSelect();
+
+            AddStep("press ctrl+shit+enter", () =>
+            {
+                InputManager.PressKey(Key.ControlLeft);
+                InputManager.PressKey(Key.ShiftLeft);
+                InputManager.PressKey(Key.Enter);
+
+                InputManager.ReleaseKey(Key.ControlLeft);
+                InputManager.ReleaseKey(Key.ShiftLeft);
+                InputManager.ReleaseKey(Key.Enter);
+            });
+
+            AddUntilStep("wait for player", () => Stack.CurrentScreen is PlayerLoader);
+
+            AddAssert("cinema enabled", () => songSelect.Mods.Value.FirstOrDefault() is ModCinema);
+
+            AddUntilStep("wait for return to ss", () => songSelect.IsCurrentScreen());
+
+            AddAssert("mod disabled", () => songSelect.Mods.Value.Count == 0);
+        }
+
+        [Test]
+        public void TestAutoplayWithCtrlShiftEnter()
+        {
+            addRulesetImportStep(0);
+
+            createSongSelect();
+
+            AddStep("select autoplay", () =>
+            {
+                songSelect.ModSelect.SelectTypes(new[] { typeof(ModAutoplay) });
+            });
+
+            AddStep("press ctrl+shit+enter", () =>
+            {
+                InputManager.PressKey(Key.ControlLeft);
+                InputManager.PressKey(Key.ShiftLeft);
+                InputManager.PressKey(Key.Enter);
+
+                InputManager.ReleaseKey(Key.ControlLeft);
+                InputManager.ReleaseKey(Key.ShiftLeft);
+                InputManager.ReleaseKey(Key.Enter);
+            });
+
+            AddUntilStep("wait for player", () => Stack.CurrentScreen is PlayerLoader);
+
+            AddAssert("only cinema enabled", () => songSelect.Mods.Value.FirstOrDefault() is ModCinema && songSelect.Mods.Value.Count == 1);
```

Comment:
Could simplify by doing
```suggestion
            AddAssert("only cinema enabled", () => songSelect.Mods.Value.SingleOrDefault() is ModCinema);
```